### PR TITLE
docs: improve display of color block in docs

### PR DESCRIPTION
**Pull request by Wxh16144**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     update blog style      |
| 🇨🇳 Chinese |     更新博客排版       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 32eebd6</samp>

This pull request enhances the blog post about color picker by adding a new component `<ColorChunk>` to show color values and blocks, and by using CSS gradients instead of images for color variations. It also synchronizes the Chinese version of the post with the English version and fixes some minor errors.

### <samp>🤖 Generated by Copilot at 32eebd6</samp>

*  Add a custom component `<ColorChunk>` to wrap the color values in different color models and render the corresponding color blocks in the blog post ([link](https://github.com/ant-design/ant-design/pull/44790/files?diff=unified&w=0#diff-d35327ba9ca95fc92311e25845e955f36653c7621dd09dfad8581c913c6d73daL17-R17), [link](https://github.com/ant-design/ant-design/pull/44790/files?diff=unified&w=0#diff-d35327ba9ca95fc92311e25845e955f36653c7621dd09dfad8581c913c6d73daL23-R27), [link](https://github.com/ant-design/ant-design/pull/44790/files?diff=unified&w=0#diff-a7fc50163332951384794ce6f1f7038704d241491d1179336aebfc3b153d9c06L17-R17), [link](https://github.com/ant-design/ant-design/pull/44790/files?diff=unified&w=0#diff-a7fc50163332951384794ce6f1f7038704d241491d1179336aebfc3b153d9c06L23-R27))
* Replace the images of the blue color with different saturation and brightness values with `<div>` elements that have the same background color and size, but use linear gradient overlays to simulate the color variation ([link](https://github.com/ant-design/ant-design/pull/44790/files?diff=unified&w=0#diff-d35327ba9ca95fc92311e25845e955f36653c7621dd09dfad8581c913c6d73daL55-R58), [link](https://github.com/ant-design/ant-design/pull/44790/files?diff=unified&w=0#diff-d35327ba9ca95fc92311e25845e955f36653c7621dd09dfad8581c913c6d73daL64-R69), [link](https://github.com/ant-design/ant-design/pull/44790/files?diff=unified&w=0#diff-d35327ba9ca95fc92311e25845e955f36653c7621dd09dfad8581c913c6d73daL73-R80), [link](https://github.com/ant-design/ant-design/pull/44790/files?diff=unified&w=0#diff-a7fc50163332951384794ce6f1f7038704d241491d1179336aebfc3b153d9c06L55-R58), [link](https://github.com/ant-design/ant-design/pull/44790/files?diff=unified&w=0#diff-a7fc50163332951384794ce6f1f7038704d241491d1179336aebfc3b153d9c06L64-R69), [link](https://github.com/ant-design/ant-design/pull/44790/files?diff=unified&w=0#diff-a7fc50163332951384794ce6f1f7038704d241491d1179336aebfc3b153d9c06L73-R80))

### CODE REVIEW

**Comment by Wxh16144** [comment]
image

（如图）这个 pr 如此改动后，在 github markdown 中预览效果就不是很好了。

也包括之前提交的 https://github.com/ant-design/ant-design/pull/39348 PR，修改后文档不那么直观。根据这篇 blog，我从 dumi-plugin 的角度封装了一个适用于 dumi2 的插件 https://github.com/Wxh16144/dumi-plugin-color-chunk#readme

想问问 @afc163  @MadCcc 有没有可能再改回 \`xxx\`?

解决方案有两个，一个是直接用我这个插件，二是将源码 copy 到 antd .dumi 自定义插件目录中？

--- 

本来想直接提交 Feature 内置到 dumi，但是考虑依赖了 `@ctrl/tinycolor` , 感觉体积会增加很多。 所以考虑在 dumi 那边添加到插件市场链接，而不内置到 dumi 中。 cc @PeachScript 

大佬们意见如何？
image

image

File: package.json [hunk 1]
```diff
@@ -217,6 +217,7 @@
     "crypto": "^1.0.1",
     "dekko": "^0.2.1",
     "dumi": "^2.1.23",
+    "dumi-plugin-color-chunk": "1.0.2",
```
Comment (afc163) [inline package.json:220]: @PeachScript 是不是可以搞个 dumi 官方插件集？

Comment (afc163) [inline package.json:220]:
```suggestion
    "dumi-plugin-color-chunk": "^1.0.2",
```

Comment (PeachScript) [reply package.json:220]: 类似 @umijs/plugins 么，感觉放一起依赖安装太大了…

**Review by RedJue — APPROVED** [review]

**Review by kiner-tang — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (afc163) [reply package.json:220]: 我意思就是个聚合页。

Comment (Wxh16144) [reply package.json:220]: 在 dumi 那边提了 pr https://github.com/umijs/dumi/pull/1890， 但是内容太少了 😭

File: .dumi/theme/builtins/ColorChunk/index.tsx [hunk 2]
```diff
@@ -27,12 +27,12 @@ const useStyle = createStyles(({ token, css }) => ({
...
   const { styles } = useStyle();
-  const { color, children } = props;
+  const { value, children } = props;
 
   const dotColor = React.useMemo(() => {
-    const _color = new TinyColor(color).toHex8String();
+    const _color = new TinyColor(value).toHex8String();
     return _color.endsWith('ff') ? _color.slice(0, -2) : _color;
-  }, [color]);
+  }, [value]);
```
Comment (MadCcc) [inline .dumi/theme/builtins/ColorChunk/index.tsx:35]: 这个 builtin 删了?

File: package.json (hunk 1 above)
Comment (afc163) [reply package.json:220]: ping

File: .dumi/theme/builtins/ColorChunk/index.tsx (hunk 2 above)
Comment (Wxh16144) [reply .dumi/theme/builtins/ColorChunk/index.tsx:35]: 不能，因为 token 的 table 用到了 https://github.com/ant-design/ant-design/blob/4d3814478c6f440dae515b7bcaed6aa246916576/.dumi/theme/builtins/TokenTable/index.tsx#L81

**Comment by Wxh16144** [comment]
ready~

**Pull request merged**